units/host_status.go

```diff
@@ -186,6 +187,11 @@ func (j *cloudHostReadyJob) setCloudHostStatus(ctx context.Context, m cloud.Mana
 		event.LogHostTerminatedExternally(h.Id, h.Status)
 
 		catcher := grip.NewBasicCatcher()
+		if h.RunningTask != "" {
+			if err := task.AddHostCreateDetails(h.RunningTask, h.Id, j.Error()); err != nil {
```

Comment:
same idea here: there's likely no error in the job that we want to log (or even if there is, it's not what we want to tell the user). We want to tell the user that the host has been terminated externally.